crates/core/tedge_actors/src/runtime.rs

```diff
@@ -493,30 +496,36 @@ mod tests {
         );
     }
 
-    // TODO: An actor panic doesn't print the way I want
-    // #[tokio::test]
-    // async fn actor_panics() {
-    //     let (mut actions_sender, mut events_receiver, runtime_request_sender, ra) = init();
-    //     let (mut input_sender, mut output_receiver, actor) =
-    //         create_actor(Panic {}, &runtime_request_sender);
-
-    //     actions_sender
-    //         .send(RuntimeAction::Spawn(Box::new(actor)))
-    //         .await
-    //         .unwrap();
-
-    //     let wait_for_messages = async {
-    //         output_receiver.next().await;
-
-    //         while let Some(event) = events_receiver.next().await {
-    //             dbg!(&event);
-    //         }
-    //         true
-    //     };
-
-    //     tokio::select! {
-    //         spawned_actor_event_received = wait_for_messages => assert!(spawned_actor_event_received, "The actor was not spawned"),
-    //         _ = ra.run() => panic!("The runtime actor finished unexpectedly")
-    //     };
-    // }
+    #[tokio::test]
+    async fn actor_panics() {
+        let (mut actions_sender, mut events_receiver, ra) = init();
+        let (_, _, panic_actor) = create_actor(Panic);
+        let (mut sender, _, echo_actor) = create_actor(Echo);
+
+        actions_sender
+            .send(RuntimeAction::Spawn(Box::new(panic_actor)))
+            .await
+            .unwrap();
+
+        actions_sender
+            .send(RuntimeAction::Spawn(Box::new(echo_actor)))
+            .await
+            .unwrap();
+
+        let wait_for_actors_to_panic = async {
+            while let Some(event) = events_receiver.next().await {
+                if matches!(event, RuntimeEvent::Aborted { .. }) {
```

Comment:
Shouldn't we match the name of the `Panic` actor as well to be sure that it is the one that failed?